esi_leap/tests/api/controllers/test_types.py

```diff
@@ -12,15 +12,35 @@
 
 from esi_leap.api.controllers import types as types
 import unittest as unittest
+from wsme import types as wtypes
+
+
+class Tester(object):
+
+    def __init__(self, name, uuid):
+        self.name = name
+        self.uuid = uuid
 
 
 class TestCollection(unittest.TestCase):
 
     def setUp(self):
         self.test_collection = types.Collection()
         self.test_collection._type = 'stuff'
-        self.test_collection.stuff = [1, 2, 3]
+        self.test_collection.stuff = []
+        self.test_collection.stuff.append(Tester('1', 'aaa'))
+        self.test_collection.stuff.append(Tester('2', 'aaaa'))
+        self.test_collection.stuff.append(Tester('3', 'aaaaa'))
+        self._type = 'url'
 
     def test_has_next(self):
         self.assertEqual(self.test_collection.has_next(3), True)
         self.assertEqual(self.test_collection.has_next(0), False)
+
+    def test_get_next(self):
+        kwargs = {"key1": "Things", "key2": "Stuff"}
+        link = ("{'key1': 'Things', 'key2': 'Stuff'}"
+                "/v1/stuff?limit=3&marker=aaaaa")
```

Comment:
Using 'aaaaa' is a little dangerous; someone might change the uuid above and not realize this test would fail. I'd suggest the following:

* In the setUp function, have "self.obj1 = mock.Mock(...)", for all three objects, and self.test_collection=[self.obj1, etc]
* Replace aaaaa here with self.obj3.uuid

Reply:
i changed it to separate objects and then replaced aaaaa for the object.uuid.